File: sdkwork-ai-sdk-csharp/Models/ThreadResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ThreadResponse
    {
        public string? Id { get; set; }
        public string? Object { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public int? CreatedAt { get; set; }
        public Dictionary<string, object>? ToolResources { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/thread_response.go
package types

// 线程响应
type ThreadResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Metadata map[string]string `json:"metadata"`
	CreatedAt int `json:"created_at"`
	ToolResources map[string]interface{} `json:"tool_resources"`
}
